src/api/spec/controllers/webui/patchinfo_controller_spec.rb

```diff
@@ -78,7 +78,7 @@ def do_proper_post_save
 
     context 'when the patchinfo package file is not found' do
       before do
-        Package.any_instance.stubs(:patchinfo).returns(nil)
+        allow_any_instance_of(Package).to receive(:patchinfo).and_return(nil)
```

Comment:
the `.and_return(nil)` is not needed, just:

`allow_any_instance_of(Package).to receive(:patchinfo)`